src/Infrastructure/Persistence/ParticipantDynamoDbRepository.cs

```diff
@@ -59,16 +55,16 @@ public async Task<ParticipantDetails> GetParticipantDetailsByEmailAsync(string e
                 var request = new ScanRequest
                 {
                     TableName = _config.OverrideTableName,
-                    FilterExpression = "#email = :email",
+                    FilterExpression = $"{filterKey} = :{filterKey}",
```

Comment:
`FilterExpression = $"#{dbCol} = :{dbCol}",`

Reply:
So dbCol is capitalised and filter key is not, is it better to do this through code?